destreamer.ts

```diff
@@ -89,7 +89,7 @@ async function DoInteractiveLogin(username?: string): Promise<Session> {
         args: ['--disable-dev-shm-usage']
     });
     const page = (await browser.pages())[0];
-    console.log('Navigating to microsoftonline.com login page...');
+    console.log('Navigating to microsoftonline.com for login procedure...');
 
     await page.goto(loginUrl, { waitUntil: "load" });

```

Comment:
missed this, we don't know the url of the user, loginUrl is "static"
@snobu

Reply:
are you suggesting that even the general login page can be different if you have a particular Microsoft user type?

Reply:
It is a wild wild world out there when it comes to identity providers and federation. Let's avoid it :) See comments below.

Reply:
Remeber Microsoft Stream is aimed at companies not consumers.. and you know the enterprise can be very, very creative with their security :)